package.json

```
@@ -55,16 +55,22 @@
     "@sentry/react": "10.5.0",
     "@sentry/tracing": "7.120.4",
     "@supabase/supabase-js": "2.54.0",
+    "@testing-library/jest-dom": "6.4.2",
```

Biome (2.1.2)

[error] The key @testing-library/jest-dom was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)